docs/Other SDL Features/Batch Sending RPCs/index.md

```diff
@@ -7,9 +7,15 @@ Both methods have optional progress and completion handlers. Use the `progressHa
 @![android,javaSE,javaEE]
 Both methods have optional listener that is specific to them, the `OnMultipleRequestListener`. This listener will provide more information than the normal `OnRPCResponseListener`.!@
 
+@![javascript]
+Both methods can have the `await` syntax be used to pause execution until all the responses return, and errors can be caught by attaching a `catch` handler. The concurrent method accepts an array of requests and will return an array of responses, while the sequential method accepts an array of requests and returns the last RPC response in the array.!@
+
 ## Sending Concurrent Requests
 When you send multiple RPCs concurrently, it will not wait for the response of the previous RPC before sending the next one. Therefore, there is no guarantee that responses will be returned in order, and you will not be able to use information sent in a previous RPC for a later RPC.
 
+@![javascript]
+Note that for the JavaScript library the concurrent method will honor the ordering of the requests passed in (the method uses Promise.all behind the scenes). Each response in the array has the same position of their matching request.!@
```

Comment:
Which method is this talking about? Is it the sdlManager's `sendRpcs`? It should probably be stated somewhere.